src/compressor/zlib/CompressionZlib.cc

```diff
@@ -0,0 +1,148 @@
+/*
+ * Ceph - scalable distributed file system
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ */
+
+// -----------------------------------------------------------------------------
+#include "common/debug.h"
+#include "CompressionZlib.h"
+#include "osd/osd_types.h"
+// -----------------------------------------------------------------------------
+
+#include <zlib.h>
+
+// -----------------------------------------------------------------------------
+#define dout_subsys ceph_subsys_osd
```

Comment:
You're right. Please note however that these changes  are from another PR #6361 incorporated here for the sake of completeness (this is valid for the whole commit 491b153). So it's probably better to resolve the issue in the original PR. Will push the process...